README.md

```diff
@@ -116,12 +99,14 @@ Instead, we expect to help organizations identify DDD knowledge gaps and use the
 
 - What's the main goal of an Event Storming? (Open text)
 
-- How does Event Storming relate with Strategic Design and Tactical Design? (Open text)
+- How does Event Storming relate to Strategic Design and Tactical Design? (Open text)
 
 ### Tactical Design
 
 - Why Hexagonal Architecture is so popular within DDD? (Open text)
 
+- Is Layered Architecture compatible with DDD? (Open text)
```

Comment:
My first reaction was "Is it?" I am not sure what the intention behind the questions is, I find it very hard to answer them.

Reply:
Thanks for the question @yellowbrickc ❤️ 

I would say most of the questions, if not all, don't have an answer that are 100% right. They all are about the grays. The intentions of the questions are to see the reflections people have regarding this topics.

Because it's a familiarty assessment, and who does the survey is also biased towards some answers. I suggest people to focus more on how people think about a topic.

Are they too rigit or too superficial? This might be a hint to dig deeper on those topics so that during a training for example people can open their mind to new ways of understanding the topics.

Does it make sense?

I'm thinking on adding this to the repository as well.

Reply:
Agree on all, there isn't a black-or-white answer to these questions (yet). My "Is it?" was mostly referring to the sentence "Why Hex. Arch. is so popular within DDD?", sorry for the confusion. I think both are good questions for the survey 👍 I just would try to leave it open instead of suggesting the answer. This is why I've asked for the intention. 
I could imagine something like

```suggestion
- Would you apply Hexagonal Architecture and DDD together. Why (or why not)?  (Open text)
- Would you apply Layered Architecture together. Why (or why not)? (Open text)
- Would you rather use some other Architecture style? Why? (Open text) 
```

Reply:
I Agree! 😄 I applied your suggestions :D